newIDE/app/yarn.lock

```diff
@@ -2305,17 +2305,17 @@
   "resolved" "https://registry.npmjs.org/@pixi/compressed-textures/-/compressed-textures-6.1.2.tgz"
   "version" "6.1.2"
 
-"@pixi/constants@6.1.2":
+"@pixi/constants@^6.0.4", "@pixi/constants@6.1.2":
```

Comment:
I see a 6.0.4 appearing, this might be a sign we're having two PixiJS versions - give a look with "yarn why" or "npm ls" @arthuro555

Reply:
🤔 
```
gdevelop-ide@0.1.0 C:\code\GDevelop\newIDE\app
├─┬ @pixi/tilemap@3.2.2
│ ├── @pixi/constants@6.1.2
│ ├─┬ @pixi/core@6.1.2
│ │ └── @pixi/constants@6.1.2 deduped
│ ├─┬ @pixi/graphics@6.1.2
│ │ ├── @pixi/constants@6.1.2 deduped
│ │ └─┬ @pixi/sprite@6.1.2
│ │   └── @pixi/constants@6.1.2 deduped
│ └─┬ @pixi/utils@6.1.2
│   └── @pixi/constants@6.1.2 deduped
└─┬ pixi.js-legacy@6.1.2
  ├─┬ @pixi/canvas-graphics@6.1.2
  │ └── @pixi/constants@6.1.2 deduped
  ├─┬ @pixi/canvas-mesh@6.1.2
  │ ├── @pixi/constants@6.1.2 deduped
  │ ├─┬ @pixi/mesh-extras@6.1.2
  │ │ └── @pixi/constants@6.1.2 deduped
  │ └─┬ @pixi/mesh@6.1.2
  │   └── @pixi/constants@6.1.2 deduped
  ├─┬ @pixi/canvas-particle-container@6.1.2
  │ └─┬ @pixi/particle-container@6.1.2
  │   └── @pixi/constants@6.1.2 deduped
  ├─┬ @pixi/canvas-renderer@6.1.2
  │ └── @pixi/constants@6.1.2 deduped
  ├─┬ @pixi/canvas-sprite-tiling@6.1.2
  │ └─┬ @pixi/sprite-tiling@6.1.2
  │   └── @pixi/constants@6.1.2 deduped
  ├─┬ @pixi/canvas-sprite@6.1.2
  │ └── @pixi/constants@6.1.2 deduped
  └─┬ pixi.js@6.1.2
    ├─┬ @pixi/compressed-textures@6.1.2
    │ └── @pixi/constants@6.1.2 deduped
    ├── @pixi/constants@6.1.2 deduped
    └─┬ @pixi/loaders@6.1.2
      └── @pixi/constants@6.1.2 deduped
```

Can't find 6.0.4 anywhere in there

Reply:
I completely regenerated the lockfile and this is gone now 👍